libgadget/hydra.c

```diff
@@ -374,10 +392,15 @@ hydro_ngbiter(
     if(Pressure_j == 0) {
         Pressure_j = PressurePred(P[other].PI, EntVarPred);
```

Comment:
Isn't it strange that PressurePred doesn't depend on Ti_Current?

Reply:
EntVarPred depends on Ti_Current...

Reply:
But you're right! It should depend on the predicted density.